jac-uk/admin
# membership display and edit

## What's included?
[An issue was raised regarding General Medical Council membership not showing correctly](https://judicial-appointments.slack.com/archives/C033NNNU5TP/p1651751762050909).
After a review of the `QualificationsAndMembership.vue` code, it appears i had missed the display and edit of certain memberships, GMC being one. 
This PR should fix to include these memberships and allow their editing by admins as per application edits.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Find an application which declares membership to GMC or other membership.
- Observe correct display and edit of given information.

## Risk - how likely is this to impact other areas?
🟠 Medium risk 

## Additional context
See above link to slack.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
